Sources/Rendering/Core/CellPicker/index.js

```diff
@@ -324,7 +364,12 @@ function vtkCellPicker(publicAPI, model) {
         weights[i] = 0.0;
       }
       const point = [];
-      minCell.evaluateLocation(minPCoords, point, weights);
+
+      if (vtkCellTypes.hasSubCells(cellType)) {
```

Comment:
Instead of cellType, shouldn't it be `minCellType` ? or `minCell.getType()` ? or `getCellType(minCell)`